changelog.d/7177.bugfix

```diff
@@ -0,0 +1 @@
+Mitigation for a bug in `_get_e2e_device_keys_txn` which broke federation for some servers.
```

Comment:
only a mitigation?

also I don't really think the reader cares which function it's in.  How about:

> Fix a bug which could cause outbound federation traffic to stop working if a client uploaded an incorrect e2e device signature.